ecllibrary/teststd/File/TestGetEspURL.ecl

```diff
@@ -0,0 +1,26 @@
+/*##############################################################################
+## HPCC SYSTEMS software Copyright (C) 2017 HPCC Systems®.  All rights reserved.
+############################################################################## */
+
+IMPORT Std.File;
+
+EXPORT TestGetEspURL := MODULE
+
+    EXPORT TestRuntime := MODULE
+        // Assumes that there is no user authentication on the test system
```

Comment:
Is this assumption going to cause problems?  Should it be using the credentials of the person running the regression suite, and if so how?

Reply:
I don't know of a good way to insert credentials short of providing attributes that can be edited, as I don't believe the credentials for the current user are available to ECL.  If they are, please let me know how they are exposed.

I'm inclined to leave this as-is, as the regression suite is (I believe) typically run on a version built explicitly for testing that it probably will not require credentials.  I can easily add the code necessary to provide the credentials and set them to an empty string if you think that will clarify things, though.

Reply:
This credentials question on the test does prompt an idea, however:  It would be a good idea to add optional arguments to the ECL GetEspURL() function for username and password, then format the returned URL accordingly.  I will make that change as well.